Traceback (most recent call last):
  File "/home/pranavsatheesh/host_galaxies/py_files/merger_file_from_sublink_trees_mod.py", line 843, in <module>
    
  File "/home/pranavsatheesh/host_galaxies/py_files/merger_file_from_sublink_trees_mod.py", line 572, in writeMergerFile
    # instantiatite MergerInfo class (walks thru merger tree)
  File "/home/pranavsatheesh/host_galaxies/py_files/merger_file_from_sublink_trees_mod.py", line 157, in __init__
    self.getMergerInfoSubtree(tree, minMassRatio=self.minMassRatio, massPartType=self.massPartType, 
  File "/home/pranavsatheesh/host_galaxies/py_files/merger_file_from_sublink_trees_mod.py", line 335, in getMergerInfoSubtree
    self = self.getMergerInfoSubtree(tree, minMassRatio=self.minMassRatio, 
  File "/home/pranavsatheesh/host_galaxies/py_files/merger_file_from_sublink_trees_mod.py", line 293, in getMergerInfoSubtree
    fp_maxpast_mass,np_maxpast_mass,fpinfallMass,npinfallMass,fp_mass_history, np_mass_history, fp_snap_history, np_snap_history = find_infall_and_maxpastmass(tree,fpIndex,npIndex)
  File "/home/pranavsatheesh/host_galaxies/py_files/merger_file_from_sublink_trees_mod.py", line 421, in find_infall_and_maxpastmass
    
UnboundLocalError: local variable 'fp_infall_mass' referenced before assignment
